amnestywebsite/humanity-theme
# Remove block restrictions

Ref: https://github.com/amnestywebsite/humanity-theme/issues/205

**Steps to test**:
1. check out this branch and run a build
2. all `core/*`, `yoast/*`, and `woocommerce/*` blocks should be available in the inserter
3. see https://github.com/amnestywebsite/wp-plugin-amnesty-branding/pull/17
